zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -140,8 +144,43 @@ void installNodeAndNpm() throws TaskRunnerException {
   }
 
   private ProxyConfig getProxyConfig() {
-    List<ProxyConfig.Proxy> proxy = new LinkedList<>();
-    return new ProxyConfig(proxy);
+    List<ProxyConfig.Proxy> proxies = new LinkedList<>();
+
+    String httpProxy = StringUtils.isBlank(System.getenv("http_proxy")) ?
+            System.getenv("HTTP_PROXY") : System.getenv("http_proxy");
+
+    String httpsProxy = StringUtils.isBlank(System.getenv("https_proxy")) ?
+            System.getenv("HTTPS_PROXY") : System.getenv("https_proxy");
+
+    try {
+      // Order matters, first tries secure proxy
+      proxies.add(generateProxy("secure", new URI(httpsProxy)));
+      proxies.add(generateProxy("insecure",new URI(httpProxy)));
```

Comment:
[ERROR] HeliumBundleFactory.java[158:44] (whitespace) WhitespaceAfter: ',' is not followed by whitespace.
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Zeppelin ........................................... SUCCESS [  3.897 s]
[INFO] Zeppelin: Interpreter .............................. SUCCESS [  9.132 s]
[INFO] Zeppelin: Zengine .................................. FAILURE [  1.260 s]